src/clj/game/cards/ice.clj

```diff
@@ -493,12 +499,14 @@
                                 (trash state side card))}]}
 
    "Datapike"
-   {:subroutines [{:msg "force the Runner to pay 2 [Credits] if able"
+   {:implementation "Encounter effect is manual"
```

Comment:
Yup. Datapike doesn't have an encounter effect :). It doesn't automatically end the run, though.